src/clientdlg.cpp

```diff
@@ -858,6 +858,16 @@ void CClientDlg::OnNumClientsChanged ( int iNewNumClients )
     SetMyWindowTitle ( iNewNumClients );
 }
 
+void CClientDlg::OnOpenGeneralSettings()
+{
+    ShowGeneralSettings ( SETTING_TAB_AUDIONET );
+}
+
+void CClientDlg::OnOpenMusicianProfileDialog()
+{
+    ShowGeneralSettings ( SETTING_TAB_USER );
+}
+
```

Comment:
And part of me wants a View->Advanced settings menu option to get to that tab...